src/themes/masschallenge/elements/header.variables

```diff
@@ -0,0 +1 @@
+@darkTextColor: #1C1F2A;
```

Comment:
This may not be necessary if the `header` is refactored per: https://github.com/masschallenge/front-end/pull/183#discussion_r338028957 (unless this color is reused in another location).